### ISSUE TITLE
Element IDs are not unique (Accessibility)

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/SearchWorks#3781]
Two or more elements on this page have the same id value. These should be unique.

[SiteImprove Issue with details](https://my2.siteimprove.com/Accessibility/1059585/NextGen/Issue/1?ruleId=3&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

This issue references the record view - specifically the current item Select input under the browse-related items section and the the Select input in the record toolbar. See attached screenshot. 

See #3780, which is also related to the Select input for the current item.

**Solution**
Add a qualifier to the select element for one or both selects. 

![Image](https://github.com/sul-dlss/SearchWorks/assets/6816624/d0e63a85-529b-433f-8662-c6a30275ab92)

**Comment by jcoyne** [comment]
I believe this is a problem that exists in Blacklight 7, but is fixed in Blacklight 8.
